spock-specs/src/test/groovy/org/spockframework/docs/interaction/MockMakerDocSpec.groovy

```diff
@@ -33,4 +35,36 @@ class MockMakerDocSpec extends Specification {
     then:
     1 * _.receive(_)
   }
+
+  def "mockito-inline"() {
+    given:
+// tag::mockito-inline[]
+    Subscriber subscriber = Mock(mockMaker: MockMakers.mockitoInline)
+// end::mockito-inline[]
+
+    when:
+    subscriber.receive("1")
+
+    then:
+    1 * _.receive(_)
+  }
+
+  @SuppressWarnings('UnnecessaryQualifiedReference')
+  def "mockito-inline additional settings serializable"() {
+    given:
+// tag::mock-serializable[]
+    Subscriber subscriber = Mock(mockMaker: MockMakers.mockitoInline {
+      mockito { org.mockito.MockSettings settings ->
+        settings.serializable()
+      }
```

Comment:
❓ Why did you add this indirection?

Reply:
To be able to add mockSettings for the Spock side in the future.
E.g. that we could configure, how Spock would use/create the Mockito-Mock.
If I would use the `org.mockito.MockSettings` directly, we could not add any settings to that interface.

I was also wondering, if I should rename the mock maker from `mockito-inline` to `mockito`, because that I am using the `inline` part, would be an implementation details, which could for example be configured with the mock settings.

Reply:
🤔 I would go with YAGNI, unless you already have a concrete plan to add something more here. If we go with your suggestion of renaming it, which is ok with me, then it looks even weirder
```
MockMakers.mockito {
  mockito {
    settings.serializable()
  }
}
```

If we really need to add something in the future, we could use a differently named method to offer the more complex config.

Reply:
>If we really need to add something in the future, we could use a differently named method to offer the more complex config.

That is a good point.
If have removed the settings indirection and renamed `mockito-inline` to `mockito`